chore(docs): aliases for dotnet/chsarp docs.

avodovnik opened the pull request:
Starting to clean-up/add docs related to the C# port, beginning with aliases for `QuerySelector` functions.

avodovnik committed:
First pass at adding aliases for $ and $eval for dotnet/chsarp docs.

pavelfeldman commented on docs/src/api/class-elementhandle.md:
```diff
@@ -59,6 +59,7 @@ ElementHandle instances can be used as an argument in [`method: Page.$eval`] and
 ## async method: ElementHandle.$
 * langs:
   - alias-python: query_selector
+  - alias-csharp: QuerySelectorAsync
```
I think a better solution would be to consistently rename it to `querySelector` in all of our mds, then alias it into `$` only in JS

avodovnik replied:
Going through that now, I *really* wish we'd use a language that would let me say "Find All References" :-)

pavelfeldman commented on docs/src/api/class-elementhandle.md:
```diff
@@ -70,6 +71,7 @@ returns `null`.
 ## async method: ElementHandle.$$
 * langs:
   - alias-python: query_selector_all
+  - alias-csharp: QuerySelectorAllAsync
```
ditto

avodovnik committed:
Flips the default query selector names around as per PR comments.
We now default to a readable name (querySelector, querySelectorAll, etc)
for relevant functions and introduces a JavaScript alias for the rest.

Additionally, this adds an exception throw when the match for option failed,
because that proved to be a faster way to resolve the problems as they show up.

yury-s committed:
Yury's rebase on top of his changes
Update javascript.md
Update python.md
match by alias
Update more inline links
Remove spaces from langs list

yury-s commented:
@avodovnik I rebased your patch on top of my changes, applied a couple more fixes and uploaded everything to https://github.com/yury-s/playwright/commits/avodovnik-dev/anvod/dotnet/docs (sorry, didn't manage to push to this PR directly, may be it's not possible), feel free to upload it here. We need to fix docs generator before committing this PR though as method names in js docs are not generated correctly (I tried with `SRC_DIR=~/playwright npm run roll`).

yury-s commented:
Okay, docs generator is fixed in https://github.com/microsoft/playwright.dev/pull/97, this change is good to go.

avodovnik commented:
@yury-s OK, I _think_ I got your changes from that branch over here, but I'd appreciate a quick cursory check.

yury-s commented on docs/src/api/class-elementhandle.md:
```diff
@@ -112,18 +115,23 @@ assert tweet_handle.eval_on_selector(".like", "node => node.innerText") == "100"
+### param: ElementHandle.evalOnSelector.selector = %%-query-selector-%%
 
-### param: ElementHandle.$eval.expression = %%-evaluate-expression-%%
+### param: ElementHandle.evalOnSelector.pageFunction
```
this should be in javascript.md, looks like bad merge

yury-s commented on docs/src/api/class-elementhandle.md:
```diff
@@ -159,11 +167,15 @@ feed_handle = page.query_selector(".feed")
-### param: ElementHandle.$$eval.selector = %%-query-selector-%%
+### param: ElementHandle.evalOnSelectorAll.selector = %%-query-selector-%%
+
+### param: ElementHandle.evalOnSelectorAll.pageFunction
```
this should be in javascript.md, looks like bad merge

yury-s committed:
revive lost changes in class-elementhandle.md

yury-s committed:
revive lost changes

yury-s committed:
restore lost changes

yury-s committed:
fix alphabetic order

yury-s committed:
fix alphabetic order

yury-s committed:
fix alphabetic order
